### ISSUE TITLE
Back-to-top button on mobile devices.

### ISSUE THREAD

**Issue by Tarocch1** [vuejs/vuepress#1220]
## Feature request

Back-to-top button on mobile devices.

in vuepress/packages/@vuepress/plugin-back-to-top/BackToTop.vue:

``` css
@media (max-width: 959px) {
  .go-to-top {
    display: none;
  }
}
```

I dont understand why.

#### What problem does this feature solve?

Show back-to-top button on mobile devices.

#### Are you willing to work on this yourself?**

Y

**Comment by ulivz** [comment]
Contribution welcome.

**Comment by Shigma** [comment]
Fixed by #1222
